.rancher-pipeline.yml

```diff
@@ -0,0 +1,27 @@
+stages:
+- name: Build
+  steps:
+  - runScriptConfig:
+      image: maven:3-openjdk-8
+      shellScript: mvn package -DskipTests
+- name: Build Docker with DIND
+  steps:
+  - publishImageConfig:
+      dockerfilePath: ./Dockerfile
+      buildContext: .
+      tag: docker.dev.folio.org/mod-circulation:firebird-latest
+      pushRemote: true
+      registry: docker.dev.folio.org
+- name: Deploy
+  steps:
+  - applyAppConfig:
+      catalogTemplate: p-ngvqv:firebird-helmcharts-mod-circulation
+      version: 0.1.26
+      answers:
+        image.repository: docker.dev.folio.org/mod-circulation
+        image.tag: firebird-latest
+        postJob.enabled: true
+      targetNamespace: firebird
+      name: mod-circulation
+timeout: 60
+notification: {}
```

Comment:
From my perspective it will be useful to have possibility to re-deploy circulation on Rancher's environment. But I'm not sure. I'm fine with removing of this file if, for example, we can use it in our own forks of mod-circulation.  @celticmask , could you please help us with this? How can we organize this process?

Reply:
> From my perspective it will be useful to have possibility to re-deploy circulation on Rancher's environment. 

I agree, that would be useful. 

However, this configuration is really specific to the team environment, not the module, as that can be used in many teams environments. To me, that means it should not be part of the module code, at least not the mainline.